source/UIAHandler/browseMode.py

```diff
@@ -486,6 +486,13 @@ def _iterNodesByType(self,nodeType,direction="next",pos=None):
 				)
 			])
 			return UIAControlQuicknavIterator(nodeType, self, pos, condition, direction)
+		elif nodeType == "tab":
+			condition = createUIAMultiPropertyCondition(

```

Comment:
Please change this to a single condition, see the case for separator as an example.